vdom/src/com/vdom/core/Player.java

```diff
@@ -24,6 +24,7 @@
     public static final String THREE_PLUS_COPY_ACTION_CARDS = "Three Plus Copy Action Cards";
     public static final String NON_VICTORY_EMPTY_SUPPLY_PILE_CARDS = "Non Victory Empty Supply Pile Cards";
     public static final String SECOND_MOST_COMMON_ACTION_CARDS = "Second Most Common Action Cards";
+    public static final String CASTLE_CARDS = "Castle Cards";
```

Comment:
I would recommend keeping the castle victory counting code as it was so that each kind of castle shows up separately in the list, not as a combined score. I'm looking for a point breakdown in the list, so this is pretty important.